<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialManager</class>
 <widget class="QWidget" name="SerialManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="scanButton">
     <property name="text">
      <string>Rescan</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <widget class="QPushButton" name="xButton">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="connectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="disconnectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
